projboard/tests/test_client.py

```diff
@@ -5,3 +5,29 @@
 def test_homepage(client):
     response = client.get("/")
     assert response.status_code == 200
+
+
+@pytest.mark.django_db
+def test_create_article_get(client):
+    response = client.get('/create_article/')
+    assert response.status_code == 200
+
+    template_names = set(tmpl.origin.template_name for tmpl in response.templates)
+    assert 'article/article.html' in template_names
+
+    # check that form has returned to the user through GET command
+    assert b"""input type="text" name="title" maxlength="1000" required id="id_title">""" in response.content
```

Comment:
You are asserting the template, line 19 is not a good practice

Reply:
Done at [98a2d9a](https://github.com/redhat-beyond/Smarticle/pull/70/commits/98a2d9af404f43b381df51438456b221a003e754)